TAS.Avalonia/App.axaml.cs

```diff
@@ -10,17 +10,20 @@
 namespace TAS.Avalonia;
 
 public partial class App : Application {
+
+    public CelesteService CelesteService = new CelesteService();
+    public DialogService DialogService = new DialogService();
+
     public override void Initialize() {
-        AvaloniaLocator.CurrentMutable.Bind<IDialogService>().ToSingleton<DialogService>();
-        AvaloniaLocator.CurrentMutable.Bind<ICelesteService>().ToSingleton<CelesteService>();
         AvaloniaXamlLoader.Load(this);
     }
 
     public override void OnFrameworkInitializationCompleted() {
         if (ApplicationLifetime is IClassicDesktopStyleApplicationLifetime desktop) {
             // Line below is needed to remove Avalonia data validation.
             // Without this line you will get duplicate validations from both Avalonia and CT
-            ExpressionObserver.DataValidators.RemoveAll(x => x is DataAnnotationsValidationPlugin);
+            // ? ExpressionObserver + DataAnnotationsValidationPlugin was made internal in 11.0.0-preview5, is this still needed or was this fixed?
```

Comment:
Didn't know this came from Avalonia. Lookign at the reason for it, I found that both Avalonia and CommunityToolkit.Mvvm add data validation. However we aren't even using CommunityToolkit.Mvvm and not even Avalonia's data validation.
Seems like it should be save to just remove since I didn't experience any issues without it.